ctfpad/urls.py

```diff
@@ -65,6 +65,7 @@
     path("challenges/<uuid:pk>/score/", views.challenges.ChallengeSetFlagView.as_view(), name="challenges-score"),
     path("challenges/<uuid:pk>/export/", views.challenges.ChallengeExportAsGithubPageView.as_view(),
          name="challenges-export"),
+    path("challenges/<uuid:pk>/workon", views.challenges.ChallengeWorkOn.as_view(), name="challenges-workon"),
```

Comment:
Can you change the name? It's actually confusing about what's going on. IMHO it's the other way round, this should be from member, that assigns himself a challenge (i.e. post="user/pk/assign" , data=current_challenge)

Also we need to make it a post request, to benefit from the csrf  protection.

That's kindda similar to the assign feature in github for issues/pr reviews/ etc.

Reply:
Ah yeah, that makes sense, will change